index.js

```diff
@@ -23,66 +28,103 @@
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red")
+
+let thirstLevel = 12;
+
+function drinkWater(thirstLevel) {
+  console.log('Man I sure am thirsty');
+  thirstLevel--
+  console.log('Ahh that hits the spot');
+  return thirstLevel
+}
+
```

Comment:
Nice work. Good use of -- and remembering that we need to explicitly return values in JS.

As an additional exercise, how would this look as an arrow function?